cypress/pages/home.page.ts

```diff
@@ -10,4 +13,14 @@ export const homePage = {
   openBankAccounts: () => {
     cy.get(homeLocator.bankAccountsButton).click();
   },
+  checkMoneyTransaction: (transactionType: String) => {
+    cy.fixture("transaction").then((transaction: any) => {
+      cy.get(homeLocator.transactionList).within(() => {
+        cy.get("li:first")
```

Comment:
It's better to move these selectors at least to variables in this file, so they could be refactored easaly.